### DISCUSSION TITLE
How can I add an already built icon to my GitHub pages website?

### DISCUSSION THREAD

**Discussion by Vladimir-Moskalenko** [community/community#32476] [Other Feature Feedback, Questions, & Ideas] [answered]
I just finished building my Github Pages website, and now want to add a favicon to it. How do I do it? I tried the basic method `<link rel="icon" type="image/x-icon" href="/images/favicon.ico">` and created a folder inside my GitHub project folder, where I placed the icon, but nothing works. Any ideas?

**Comment by radar-blip** [comment]
Take the first slash out of `href="/images/favicon.ico">` in front of images.

**Comment by jfullstackdev** [comment]
try 

from `href="/images/favicon.ico"` to `href="images/favicon.ico"`

**Comment by Vladimir-Moskalenko** [comment, marked as answer]
Surprisingly, this method made the icon visible on the name.github.io/index.html. I was checking name.github.io/
